src/panels/lovelace/cards/hui-alarm-panel-card.ts

```diff
@@ -81,6 +82,13 @@ class HuiAlarmPanelCard extends LitElement implements LovelaceCard {
     this._code = "";
   }
 
+  protected updated(changedProperties: PropertyValues): void {
+    super.updated(changedProperties);
+    if (this.hass && this._config) {
```

Comment:
We should also check if the theme is changed:
```
    const oldHass = changedProps.get("hass") as HomeAssistant | undefined;
    if (!oldHass || oldHass.themes !== this.hass.themes) {
```

Reply:
Ohw and `shouldUpdate` should be updated, it won't update now on theme changes.

Reply:
I didn't know you could add themes on the fly

Reply:
`frontend.reload_themes`

Reply:
🤦‍♀️
I'll get that add to the other cards as well as I don't think they have that either.